app/helpers/chat_channel_memberships_helper.rb

```diff
@@ -15,4 +15,30 @@ def formatted_membership_user(memberships)
       }
     end
   end
+
+  def membership_users(memberships)
+    memberships.includes(:user).map do |membership|
+      {
```

Comment:
Shouldn't this block use the `format_membership` helper method instead of duplicating everything?